eng/versioning/external_dependencies.txt

```diff
@@ -0,0 +1,221 @@
+# Format;
+# groupId:artifactId;dependency-version
+com.fasterxml.jackson.core:jackson-annotations;2.10.0
+com.fasterxml.jackson.core:jackson-core;2.10.0
+com.fasterxml.jackson.core:jackson-databind;2.10.0
+com.fasterxml.jackson.dataformat:jackson-dataformat-xml;2.10.0
+com.fasterxml.jackson.datatype:jackson-datatype-jsr310;2.10.0
+com.fasterxml.uuid:java-uuid-generator;3.2.0
+com.github.spotbugs:spotbugs;4.0.0-beta3
+com.github.spotbugs:spotbugs-maven-plugin;3.1.12.2
+com.google.code.gson:gson;2.8.5
+com.google.guava:guava;24.1.1-jre
+com.microsoft.azure:azure-annotations;1.7.0
+com.microsoft.azure:azure-arm-client-runtime;1.6.15
+com.microsoft.azure:azure-client-authentication;1.6.15
+com.microsoft.azure:azure-client-runtime;1.6.15
+com.microsoft.azure:azure-core;0.9.8
+com.microsoft.azure:azure-keyvault-cryptography;1.2.2
+com.microsoft.azure:azure-mgmt-keyvault;1.11.1
+com.microsoft.azure:client-runtime;1.6.15
+com.microsoft.azure:qpid-proton-j-extensions;1.2.1
+com.microsoft.azure:spotbugs-maven-plugin;1.2.1
+com.microsoft.rest:client-runtime;1.6.15
+com.microsoft.rest.v2:client-runtime;2.1.1
+com.squareup.okhttp3:okhttp;4.2.2
+commons-codec:commons-codec;1.13
+io.micrometer:micrometer-core;1.2.0
+io.micrometer:micrometer-registry-azure-monitor;1.2.0
+io.micrometer:micrometer-registry-graphite;1.2.0
+io.netty:netty-buffer;4.1.42.Final
+io.netty:netty-codec-http;4.1.42.Final
+io.netty:netty-handler;4.1.42.Final
+io.netty:netty-handler-proxy;4.1.42.Final
+io.netty:netty-tcnative;2.0.26.Final
+io.netty:netty-tcnative-boringssl-static;2.0.26.Final
+io.opencensus:opencensus-api;0.24.0
+io.opencensus:opencensus-impl;0.24.0
+# SNAPSHOTS should not be allowed
```

Comment:
Should it be good to remove this comment now?

Reply:
Doh! Yes @samvaity, you are correct. I've removed the comment locally and it'll be updated when I push the next commit.